samples/snippets/src/main/java/pubsub/SubscribeSyncExample.java

```diff
@@ -60,12 +60,20 @@ public static void subscribeSyncExample(
 
       // Use pullCallable().futureCall to asynchronously perform this operation.
       PullResponse pullResponse = subscriber.pullCallable().call(pullRequest);
+
+      // Exit the program if the pull response is empty.
+      if (pullResponse.getReceivedMessagesList().isEmpty()) {
+        System.out.println("No message was pulled. Exiting.");
+        System.exit(0);
```

Comment:
System.exit probably isn't what you want here (it may crash the tests) - don't you just want to return?

Reply:
You're right. Updated to return.